sharding/indexHashedNodesCoordinatorWithRater.go

```diff
@@ -0,0 +1,52 @@
+package sharding
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core/check"
+)
+
+type indexHashedNodesCoordinatorWithRater struct {
+	*indexHashedNodesCoordinator
+	RatingReader
+}
+
+// NewIndexHashedNodesCoordinator creates a new index hashed group selector
+func NewIndexHashedNodesCoordinatorWithRater(
+	indexNodesCoordinator *indexHashedNodesCoordinator,
+	rater RatingReader,
+) (*indexHashedNodesCoordinatorWithRater,
```

Comment:
output parameters should stay in the same line.

Reply:
fixed